src/getRepositories.js

```diff
@@ -83,8 +83,9 @@ function generateQuery(endCursor, { f }) {
 
 module.exports = async function getRepositories(flags, filter) {
 	// Get all repositories
+  console.log('Loading repositories...');
 	const { points, repositories } = await getRepos(generateQuery, flags, filter);
-
+  console.log('Loaded');
```

Comment:
this seems a bit useless, except to "close out" the loading statement, since the table is rendering.

Reply:
@abdumamdouh remove this line